build.sbt

```diff
@@ -176,16 +182,15 @@ lazy val V = new {
   val scalameta = "4.3.24"
   val semanticdb = scalameta
   val bsp = "2.0.0-M13"
-  val bloop = "1.4.4-25-7e876299"
-  val scala3 = "0.26.0"
-  val scala3Candidate = "0.27.0-RC1"
+  val bloop = "1.4.4-29-86b60939"
```

Comment:
We plan to do a proper release, so this will be switched to `1.4.5`.

Reply:
Also, testing this out, I'm actually unable to run bloop on the generated output for mtag3
```
~/Documents/scala-workspace/scalameta-org/metals update-dotty 7m 16s
❯ bloop projects
[E] Failed to load project from /Users/ckipp/Documents/scala-workspace/scalameta-org/metals/.bloop/mtags3-test.json
bloop.data.Project$ProjectReadException: Failed to load project from /Users/ckipp/Documents/scala-workspace/scalameta-org/metals/.bloop/mtags3.json
        at bloop.data.Project$.fromBytesAndOrigin(Project.scala:216)⏎                                  
~/Documents/scala-workspace/scalameta-org/metals update-dotty
❯ bloop about
bloop v1.4.4-25-7e876299

Using Scala v2.12.8 and Zinc v1.3.0-M4+45-d4354be3
Running on Java JDK v1.8.0_242 (/Users/ckipp/.sdkman/candidates/java/8.0.242-librca/jre)
  -> Supports debugging user code, Java Debug Interface (JDI) is available.
Maintained by the Scala Center (Jorge Vicente Cantero, Martin Duhem)
```

Reply:
You might need the newest Bloop to make it work, so probably run restart for Bloop. This is also one of the reasons I want to release 1.4.5 bloop version. It will be nicely reported then.

Reply:
Whoops, I though I was on the latest, I looked at the last `9` and assumed it was the same without seeing the `39` vs `99`. My bad!